Ladybird/Qt/BrowserWindow.h

```diff
@@ -212,6 +214,7 @@ public slots:
     StringView m_webdriver_content_ipc_path;
 
     bool m_allow_popups { false };
+    QToolBar* m_new_tab_button_toolbar { nullptr };
```

Comment:
A bit of an awkward place to put this, it is up to you, but maybe it could be below `m_zoom menu` like so?
```cpp
    QMenu* m_zoom_menu { nullptr };

    QToolBar* m_new_tab_button_toolbar { nullptr };

    QMenu* m_hamburger_menu { nullptr };
```